testing-modules/junit-5/src/test/java/com/baeldung/execution/time/RandomNumberUnitTest.java

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.execution.time;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class RandomNumberUnitTest {
+
+    @Test
+    public void whenTwoRandomNumberThenAssertUniqueness() {
```

Comment:
As these are JUnit5 tests, by convention we remove the `public` keyword from both the class and the method.

There's no guarantee that two random numbers are different. This will result in a flickering test.

Given this is an article about timing tests, how about we, instead, put a `Thread.sleep` in the example. You CAN also guarantee uniqueness when generating two UUIDs.